### PR TITLE
Add more hit categories to flight pax

### LINKED ISSUE #402: FlightPax - missing hit type columns

### PR DESCRIPTION
**Pull request by tjwdev**
Sister front-end PR to "add_more_hitCategories_to_flightPax" on GTAS repo.
resolves #402 

outstanding issue: need to organize aggregate flags to space appropriately into their "columns" if alone.

### CODE REVIEW

**Comment by gtasautumn** [comment]
Couple of things. We are not counting all hit types the same everywhere. I have a flight that shows 2 hits on the Flights page:
hitsflight

But when you click to **flightpax**, it shows there are 3 hits on the **All** tab (2 are rules and one is manual):
hitsflightpaxall

Then on the **flightpax Hits** tab, the manual hit disappears
hitsflightpaxhits


We need to fix:
- **FlightPax** hit counts must match between the **All** and **Hits** tabs
- Hits on the **FlightPax** page should match those on the **Flights** page
- **Flights** page **All** tab needs the same aggregation structure as the **FlightPax** page

**Review by gtasautumn — CHANGES_REQUESTED** [review]
FlightPax hit counts must match between the All and Hits tabs
Hits on the FlightPax page should match those on the Flights page
Flights page All tab needs the same aggregation structure as the FlightPax page

File: src/pages/flightPax/FlightPax.js [hunk 1]
```diff
@@ -41,10 +41,12 @@ const FlightPax = props => {
       item.dobAge = `${alt(localeDateOnly(item.dobStr))} ${item.age}`;
       item.rulehit = item.onRuleHitList ? 1 : "";
       item.watchhit = item.onWatchList ? 1 : "";
+      item.hitCounts = `${item.lowPrioHitCount || 0}${item.medPrioHitCount ||
```
Comment (gtasautumn) [inline src/pages/flightPax/FlightPax.js:44]: hitCounts is the structure I talked about that supports sorting the severity counts by place value.  We display the data from aggregateHitsCount as before, but use hitCounts as the accessor.

**Comment by tjwdev** [comment]
Adding same aggregrate structure for flights will require more changes to other tables that I think I am wary of doing. Flights do not have direct access to the hits like Passengers do, and I do have a thought on how we can do it, by attaching the priority count to the existing flight_hit_<hit-type> tables,  but wanted it to run by people first. Otherwise, flights/flightpax with the latest push now both SHOULD count and display the same numbers AND types of hits.

File: src/pages/flights/Flights.js [hunk 2]
```diff
@@ -57,6 +57,14 @@ const Flights = props => {
       if(item.externalHitCount === 0) {item.externalHitCount=""};
       if(item.manualHitCount === 0) {item.manualHitCount=""};
 
+      item.hitCounts = `${item.lowPrioHitCount || 0}${item.medPrioHitCount ||
+      0}${item.highPrioHitCount || 0}`;
+      item.aggregateHitsCount = {
+        low: item.highPrioHitCount,
```
Comment (gtasautumn) [inline src/pages/flights/Flights.js:63]: this looks backward. Actually, it's backward on line 60 as well. I think the lows were showing up with the red color and severe ones were yellow. Meant to fix that earlier.

**Comment by gtasautumn** [comment]
For the same flight, I created a manual hit on a passengers and a watchlist hit on a third:
- Both flights tabs show the expected results
- FlightPax All tab shows the expected results
- **FlightPax Hits shows only the watchlist hits passenger. The manual hit passengers do not appear**
- **FlightPax Hits watchlist passenger has 3 hits shown when there are only 2 (doc and passenger wl hits)**

fpaxall

fpaxhits

paxdetailhits

**Comment by tjwdev** [comment]
Newest backend/frontend commits should resolve current counting problems.

**Comment by gtasautumn** [comment]
watchlist hits still don't match on the flightpax tabs.

**Pull request merged**